sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md

```diff
@@ -32,7 +32,7 @@ license-header: MICROSOFT_MIT_SMALL
 generate-client-as-impl: true
 context-client-method-parameter: true
 models-subpackage: systemevents
-customization-class: EventGridCustomization
+customization-class: src/main/java/EventGridCustomization.java
```

Comment:
Should look into upgrading the `use` version to the latest of `4.1.17` and add some of the newer features such as 

`generic-response-type: true`
`disable-client-builder: true`

Reply:
Will look into it, but not here.